bootstrap5-demo-with-translate/README.md

```diff
@@ -0,0 +1,15 @@
+## Bootstrap 4 
```

Comment:
I guess that should be `Bootstrap 5` everywhere BS4 is mentioned right?

Reply:
Fixed in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/14181c39cef77434ce50556d6dca53ab9feee981